exercises/practice/matching-brackets/matching-brackets.rkt

```diff
@@ -0,0 +1,6 @@
+#lang racket
+
+(provide balanced?)
+
+(define (balanced? str)
+  (error "Not implemented yet"))
```

Comment:
Great idea to provide a stub method implementation. Just in case you'd like to do some more PRs: a lot of exercises don't yet have a stub method defined.